vpc/globalgc/globalgc.go

```diff
@@ -115,20 +116,21 @@ func fetchDisconnectedENIs(parentCtx *context.VPCContext, vpcID string, networkI
 // TODO: Find fatal errors and return errors
 func cleanupENI(parentCtx *context.VPCContext, svc *ec2.EC2, detachTime time.Duration, eni *ec2.NetworkInterface) error {
 	ctx := parentCtx.WithField("NetworkInterfaceId", *eni.NetworkInterfaceId)
+	logger := ctx.Logger.WithField("eni-name", *eni.NetworkInterfaceId)
```

Comment:
If you use the context logger, you shouldn't have to do this.

Reply:
Nice, OK - I didn't realize that doing a `ctx.WithField()` also added fields to the logger.